src/utils/constants.js

```diff
@@ -62,3 +62,7 @@ export const profileAddCardForm = addCardModal.querySelector(".modal__form");
 
```

Comment:
**Could be improved**


We can get any form using their **unique** attribute `name` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms